components/datadog/agent/ecs.go

```diff
@@ -116,6 +116,10 @@ func ecsLinuxAgentSingleContainerDefinition(e config.CommonEnvironment, apiKeySS
 				Name:  pulumi.StringPtr("DD_PROCESS_AGENT_ENABLED"),
 				Value: pulumi.StringPtr("true"),
 			},
+			ecs.TaskDefinitionKeyValuePairArgs{
+				Name:  pulumi.StringPtr("DD_SYSTEM_PROBE_NETWORK_ENABLED"),
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Should this be an option ? Not sure if we want to enable it by default

Reply:
At the moment we have no option passing through for ECS, but I agreed would be better to have the same options than others environments.

Reply:
I added ecsagentparams setting (for example) environment variables to the agent